packages/execution-environments/src/common/endowments/index.test.ts

```diff
@@ -104,6 +102,7 @@ describe('Endowment utils', () => {
       const { endowments } = createEndowments(mockWallet as any, [
         'Buffer',
```

Comment:
Unrelated, but can we remove `Buffer` from these tests now? I must have forgotten to do so.